test/async-hooks/test-httpparser.response.js

```diff
@@ -19,11 +19,8 @@ const hooks = initHooks();
 hooks.enable();
 
 const request = Buffer.from(
-  'HTTP/1.1 200 OK' + CRLF +
-  'Content-types: text/plain' + CRLF +
-  'Content-Length: 4' + CRLF +
-  CRLF +
-  'pong'
+  `HTTP/1.1 200 OK${CRLF}Content-types: text/plain${CRLF}` +
+  `Content-Length: 4${CRLF}${CRLF}pong`
```

Comment:
Another possibility might be to use `Array.prototype.join()` but I'm not sure it's more readable:

```js
const request = Buffer.from(
  ['HTTP 1.1 200 OK',
   'Content-types: text/plain',
   'Content-Length: 4',
   '',
   'pong'
  ].join(CRLF)
)
```